aviary/docs/user_guide/hamilton_standard.md

```diff
@@ -0,0 +1,101 @@
+
+The inputs are grouped in three aspects:
+
+Geometric inputs:
+- Propeller diameter
+- Activity factor per blade (range: 80 to 200, baseline: 150)
+- Number of Blades (range: 2 to 8)
+
+Power inputs:
+- Shaft Horsepower to Propeller (hp)
+- Installation Loss Factor (0 to 1)
+
+Performance inputs:
+- Operating Altitude, ft
+- True Airspeed, knots
+- Propeller Tip Speed (Usually < 800 ft/s) 
+- Integrated Lift Coefficient (range: 0.3 to 0.8, baseline: 0.5)
+
+Note that some of the inputs are good for limited ranges. The Hamilton Standard model can have odd number of blades although the data provided are based on even number of blades. For odd number of blades, interpolations using 2, 4, 6 and 8 blade data are used. The corresponding outputs are:
+
+Geometric outputs:
+- Design Blade Pitch Angle (@0.75 Radius)
+
+Power outputs:
+- Installation Loss Factor
+- Tip Compressibility Loss Factor
+- Power Coefficient
+- Thrust Coefficient (rho=const, no losses)
+
+Performance outputs:
+- Flight Mach number
+- Propeller Tip Mach number
+- Advance Ratio
+- Tip Compressibility loss factor
+- Thrust
+- Propeller Efficiency with compressibility losses
+- Propeller Efficiency with compressibility and Installation losses
+
+As shown in the above XDSM diagram, the model is an OpenMDAO group that is composed of four components and one subgroup: 
+
+- `USatmos`
+- `PreHamiltonStandard`
+- `HamiltonStandard`
+- `InstallLoss`
+- `PostHamiltonStandard`. 
+
+`USatmos` component provides the flight condition. The flight condition is passed to `PreHamiltonStandard` component from which flight Mach number, propeller tip Mach number, advance ratio and power coefficient are computed. They are fed into `HamiltonStandard` component.
+
+![CP and CT matching](images/CPE_CTE_matching.png)
+
+HamiltonStandard is the core of the model. Given the power coefficient (CP) and advance ratio (J), it finds the blade angle (BL) by a CP-BL chart by tracing advance ratio and then with the blade angle, it finds the thrust coefficient (CT) using its CT-BL chart by tracing advance ratio again. This algorithm is shown in the above pair of charts. The CP → BL → CT chart matching algorithm is based on baseline data. If user inputs are not on baseline, it will first convert them to those baseline parameters by a sequence of interpolations to do corrections. The newly converted parameters are called “effective parameters” (e.g., CPE and CTE). The outputs are blade angle, thrust coefficient and tip compressibility loss factor.
+
+Finally, the thrust is computed in `PostHamiltonStandard` component based on thrust coefficient and tip compressibility loss factor.
+
+Hamilton Standard model uses a bunch of wind tunnel test data from un-installed propellers. When a cell is mounted around the propeller, an installation loss factor is introduced. The installation loss factor can be given by user or computed. If it is computed, we need another group of components as shown below:
```

Comment:
```suggestion
The Hamilton Standard model uses wind tunnel test data from un-installed propellers. When a nacelle is mounted behind the propeller, an installation loss factor is introduced. The installation loss factor can be given by the user or computed. If it is computed, we need another group of components as shown below:
```

Reply:
Thanks for the modification. All are accepted.